llvm/include/llvm/Analysis/TargetTransformInfo.h

```diff
@@ -22,6 +22,8 @@
 #define LLVM_ANALYSIS_TARGETTRANSFORMINFO_H
 
 #include "llvm/ADT/APInt.h"
+#include "llvm/ADT/ArrayRef.h"
+#include "llvm/ADT/SmallVector.h"
```

Comment:
Do you actually need these includes here or the user of the header file already have these includes?

Reply:
for some users, thats redundant. Will remove redundant usages. Thanks.